pkg/deploy/controller/deployerpod/controller.go

```diff
@@ -28,7 +30,7 @@ type DeployerPodController struct {
 // transientError is an error which will be retried indefinitely.
 type transientError string
 
-func (e transientError) Error() string { return "transient error handling deployer pod: " + string(e) }
+func (e transientError) Error() string { return "test error handling deployer pod: " + string(e) }
```

Comment:
transient errors may happen in any kind of deployment (test or not), I think this should stay as is (a "transient error" is much more clear than a "test error")